.github/workflows/linux_jdk17.yml

```diff
@@ -1,53 +0,0 @@
-name: Linux JDK 17 GitHub CI
```

Comment:
The original reason to have a separate workflow was to allow restarting it in isolation... when we did it, all the workflows in the same action had to be restarted, if one failed. Not sure if that is no longer the case.... [pheraps not](https://stackoverflow.com/questions/71315332/only-re-run-the-failed-job-in-a-github-actions-matrix).

Reply:
Matrix jobs have cone a long way, it is now possible to restart a single job from a workflow